selectrionsort.py

```diff
@@ -0,0 +1,15 @@
+print("Enter number of elements")
+N=int(input())
+print("Enter array")
+A=input().split(' ')
+for i in range(0,N):

```

Comment:
Real time implementation should be inside a method, name it as "selectionSort()"